components/relevancy/src/schema.rs

```diff
@@ -30,6 +30,15 @@ pub const SQL: &str = "
         count INTEGER NOT NULL,
         PRIMARY KEY (kind, interest_code)
     ) WITHOUT ROWID;
+    CREATE TABLE multi_armed_bandit(
+        bandit TEXT NOT NULL,
+        arm TEXT NOT NULL,
+        alpha INTEGER NOT NULL,
+        beta INTEGER NOT NULL,
+        clicks INTEGER NOT NULL,
+        impressions INTEGER NOT NULL,
```

Comment:
Right now, I think  `clicks` and `impressions` can always be calculated from `alpha` and `beta`, but I still like storing these in a separate column since they could diverge in the future.  For example, I'm still hoping that there could be a way to pass CTR guesses to the init function that would initialize `alpha` and `beta` to something other than 1, 1.  In that case, these columns wouldn't be redundant and it would be nice to track the actual CTR separately.